<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Flaggle_game</class>
 <widget class="QMainWindow" name="Flaggle_game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 13pt &quot;BM Jua&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="gray1_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>93</y>
      <width>75</width>
      <height>67</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gray1_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>93</y>
      <width>75</width>
      <height>67</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="flag">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>93</y>
      <width>300</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="submitButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>390</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 20pt;
font: bold;
background: rgb(187, 226, 236);
border: 1px rgb(187, 226, 236);
border-radius: 12px;
letter-spacing:1px</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>461</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0)</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="ans1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="ans2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="ans3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="ans4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="ans5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>250</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="ans1_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="ans2_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="ans3_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>150</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="ans4_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>200</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="ans5_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>250</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="AlreadyGuessed">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;BM Jua&quot;;</string>
     </property>
     <property name="text">
      <string>You have guessed 0 countries</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="gray3_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>227</y>
      <width>75</width>
      <height>66</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gray1_3">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>93</y>
      <width>75</width>
      <height>67</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gray1_1">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>93</y>
      <width>75</width>
      <height>67</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gray3_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>227</y>
      <width>75</width>
      <height>66</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gray3_1">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>227</y>
      <width>75</width>
      <height>66</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gray2_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>75</width>
      <height>67</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gray3_3">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>227</y>
      <width>75</width>
      <height>66</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gray2_1">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>160</y>
      <width>75</width>
      <height>67</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gray2_3">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>160</y>
      <width>75</width>
      <height>67</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="guessFlagcomboBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>340</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="gray2_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>75</width>
      <height>67</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>20</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none;
border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/image for c++ project/Button/settingGearButton.png</normaloff>:/image for c++ project/Button/settingGearButton.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 30pt &quot;BM Jua&quot;;
color: rgb(0,0,0)</string>
    </property>
    <property name="text">
     <string>Score: 10</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
